@navikt/ds-react/copy-to-clipboard/src/CopyToClipboard.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import { forwardRef, HTMLAttributes, useEffect, useRef, useState } from "react";
+import { Files } from "@navikt/ds-icons";
+import "@navikt/ds-css/baseline/utility.css";
+import "./style.css";
+
+export interface CopyToClipboardProps
+  extends HTMLAttributes<HTMLButtonElement> {
+  /** Verdi som skal kopieres */
+  value: string;
+  /** Beskrivelse av verdien, feks "personnummer", "navn", "epost" etc. */
+  label: string;
```

Comment:
Kan det være en ide å la bruker spesifisere `title` og `success`-melding selv, for å gi større fleksibilitet? Det er jo ikke sikkert at teamet ønsker tittel med template `Kopier ${label} (${value})`